src/pages/UserProfile/MyAccount.jsx

```diff
@@ -42,30 +53,29 @@ function MyAccount() {
   return (
     <section name="myaccount" className="bg-sky-300 lg:p-20 max-lg:p-5">
       <div className="bg-white rounded-3xl">
-        <div className="lg:p-28 lg:pb-10 max-lg:p-3">
-          <div
-            className="flex flex-col items-center lg:pb-20 max-lg:p-10"
-            onClick={() => navigate('/myaccount/myaccountdetails')}
-          >
-            {/* this part navigates user to MyAccountDetails form which does not have root yet  */}
+        <div className="lg:p-16 lg:pb-8 max-lg:p-3">
+          <div className="flex flex-col items-center lg:pb-5 max-lg:p-10">
             <img
               className="m-auto h-40 w-40 rounded-full"
+              onClick={() => navigate('/myaccount/myaccountdetails')}
               src={user.photoURL ? user.photoURL : UserPhoto}
             />
             <h2 className="p-5 font-bold text-lg">
-              {user.username}
-              {user.usersurname}
+              {user.username} {user.usersurname}
             </h2>
           </div>
           <div>
-            <div className="p-5 max-lg:pr-4">
-              {/* CHITURCA will change this part workon progress */}
+            <div className="pb-5 max-lg:pr-4">
               <Slider {...settings}>
                 {blogs
-                  .filter((blog) => blog.data.userID === user.id)
-                  .map((blog, i) => {
-                    return <MyAccountCard key={i} data={blog.data} />;
-                  })}
+                  ? blogs
+                      .filter((blog) => blog.data.author.authorId === user.id)
+                      .map((blog, i) => {
+                        return <BlogCard key={i} blog={blog} />;
+                      })
+                  : emptyCard.map((holder) => (
```

Comment:
I couldn't understand this part

Reply:
To sum up,
*If user has blogs, show their blog data (which connects to BlogCard.jsx) 
*If thats not the case (i mean if user has no blog) show another, empty card to user so she/he can click on that card and start to write in WriteBlog.  And, they could know that their own blogs will be shown in that part of the MyAccount component, you know? 

I mean a new user will not know about our site structure so I tried to do something like this.
So this way I thought that MyAccount page would have been seen fully from the user's side